system/session/drivers/cookie.php

```diff
@@ -47,7 +48,7 @@ public function write($id, $cargo) {
 		$serialized = serialize($cargo);
 
 		// create a signature to verify content when unpacking
-		$sign = hash('md5', $serialized);
+		$sign = hash_hmac('sha256', $serialized, $session_key, true);
```

Comment:
Even though it is stronger, wouldn't this cause conflicts because we're starting to use a different algorithm?
(Pseudo)
`(hash('md5', 'qweasd') === hash('sha256', 'qweasd')) = False`